data/apps.ts

```diff
@@ -25,6 +25,7 @@ import tuba from "../public/apps/tuba.png"
 import mona from "../public/apps/mona.png"
 import fedica from "../public/apps/fedica.png"
 import type { StaticImageData } from "next/image"
+import focus from "../public/apps/focus.png"
```

Comment:
please move this to the end of the list of app icons, above the `import type { StaticImageData ...` line

Reply:
changed , please merge it